Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,132 @@ Meaning of columns:
+| `IfcSectionedSolidHorizontal`         | :heavy_check_mark:   | `SolidModelConverter`      | :heavy_check_mark:    | [#356](https://github.com/tumcms/Open-Infra-Platform/issues/356)|
+| `IfcSectionedSpine`                   | :x:                  | `RepresentationConverter`  | -                     | -     |
+| `IfcSegmentedReferenceCurve`          | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcSeriesParameterCurve`             | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcShellBasedSurfaceModel`           | :heavy_check_mark:   | `FaceConverter`            | :x:                   | -     |
+| `IfcSphere`                           | :heavy_check_mark:   | `SolidModelConverter`      | :x:                   | -     |
+| `IfcSphericalSurface`                 | :heavy_check_mark:   | `FaceConverter`            | :x:                   | [#150](https://github.com/tumcms/Open-Infra-Platform/issues/150)|
+| `IfcStyledItem`                       | :heavy_check_mark:   | `RepresentationConverter`  | :x:                   | -     |
+| `IfcSubedge`                          | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcSurfaceCurve`                     | :heavy_check_mark:   | `CurveConverter`           | :x:                   | -     |
+| `IfcSurfaceCurveSweptAreaSolid`       | :heavy_check_mark:   | `SolidModelConverter`      | :x:                   | -     |
+| `IfcSurfaceOfLinearExtrusion`         | :heavy_check_mark:   | `FaceConverter`            | :x:                   | [#150](https://github.com/tumcms/Open-Infra-Platform/issues/150)|
+| `IfcSurfaceOfRevolution`              | :heavy_check_mark:   | `FaceConverter`            | :x:                   | [#150](https://github.com/tumcms/Open-Infra-Platform/issues/150)|
+| `IfcSweptDiskSolid`                   | :heavy_check_mark:   | `SolidModelConverter`      | :heavy_check_mark:    | [#232](https://github.com/tumcms/Open-Infra-Platform/issues/232)|
+| `IfcSweptDiskSolidPolygonal`          | :x:                  | `SolidModelConverter`      | -                     | -     |
+| `IfcTextLiteral`                      | :x:                  | `RepresentationConverter`  | -                     | -     |
+| `IfcTextLiteralWithExtent`            | :x:                  | -                          | -                     | -     |
+| `IfcToroidalSurface`                  | :heavy_check_mark:   | `FaceConverter`            | :x:                   | [#150](https://github.com/tumcms/Open-Infra-Platform/issues/150)|
+| `IfcTransitionCurveSegment2D`         | :heavy_check_mark:   | `CurveConverter`           | :heavy_check_mark:    | -     |
+| `IfcTriangulatedIrregularNetwork`     | :heavy_check_mark:   | `FaceConverter`            | :x:                   | -     |
```

Comment:
There should be unit tests / issues for this.

Reply:
Unfortunately I did not find a suitable one...
Many files use IfcTriangulatedFaceSet

Reply:
Take #79 . From the looks of it, you can take it as your next task and produce the unit test for it.

Reply:
Resolved a81277e